runtime/src/main/scala/akka/grpc/GrpcClientSettings.scala

```diff
@@ -21,50 +21,82 @@ import scala.concurrent.duration.{ Duration, _ }
 
 object GrpcClientSettings {
 
-  /** Scala API */
-  def apply(host: String, port: Int): GrpcClientSettings =
-    // hardcoded doesn't use the resolve timeout
-    new GrpcClientSettings(host, hardcodedServiceDiscovery(host, port), port, 1.second)
+  /**
+   * Scala API
+   *
+   * Create a client hat uses a static host and port. Default configuration
+   * is loaded from reference.conf
+   */
+  def apply(host: String, port: Int)(implicit actorSystem: ActorSystem): GrpcClientSettings = {
```

Comment:
I wonder if it would make sense not to overload `apply`/`create` but create more descriptive names here. IMO that would make the API better readable and also help with discovering the different alternatives for connecting to services.

E.g. `GrpcClientSettings.connectToServiceAt(host, port)`, `GrpcClientSettings.discoverService(name)`

WDYT?

Reply:
I was thinking about this as well, I prefer it. Will update.

Reply:
sounds nice to me, leaving final call to @chbatey